### ISSUE TITLE
Hard coding clojure-mode for cljs will cause problems

### ISSUE THREAD

**Issue by arichiardi** [overtone/emacs-live#228]
I have had [this](https://github.com/clojure-emacs/cider/issues/1611) problem while updating `cider` to `0.11.0` (you can see that I am keeping up with the updates, but not maintaining publicly [this here](https://github.com/arichiardi/emacs-live/tree/ar-stable)) because of the following line:

https://github.com/overtone/emacs-live/blob/7983bc7561ea05d7432e03cc616fbfadafdf241b/packs/dev/clojure-pack/config/clojure-conf.el#L58-L59

The two lines will need to be removed, I can provide a patch when Emacs Live moves to `0.11.0` as well.

**Comment by samaaron** [comment]
Please do provide a patch :-)

**Comment by arichiardi** [comment]
Of course, but it does make sense now, I will wait the bump to `0.11.0`

**Comment by arichiardi** [comment]
Ah you mean a patch for `0.11.0`? I was thinking about it, will try to find time ok?

**Comment by Malabarba** [comment]
For the record, there's no reason to wait for cider 0.11.0 (which is already out anyway, btw). Clojure-mode has had a `clojurescript-mode` for a while now, and that's what people should be using for clojurescript files.

**Comment by arichiardi** [comment]
For the record, I did not know :D I will patch!

**Comment by Malabarba** [comment]
:wink: :+1:

**Comment by arichiardi** [comment]
Done, and btw @samaaron in the process of upgrading the `clojure-pack`, `clj-refactor` had some additional modules to add, I think I worked already on this [here](https://github.com/arichiardi/emacs-live/commits/ar-stable) so feel free to have a look at:

- https://github.com/arichiardi/emacs-live/commit/3515ec41d53b6f7a4b4170f054a8ff08cd779686
- https://github.com/arichiardi/emacs-live/commit/ea9c45918391186312444b64322bae1d530c09fa

I know I should rebase to align myself to master but I think I will do when emacs-live switches to cider `0.11.0` :D
